### ISSUE TITLE
Weekly reflection frequency

### ISSUE THREAD

**Issue by aversluis1** [PerfectFit-project/testing-tickets#71]
It has been a long time since I have had a weekly reflection dialog. I tried to scroll back and haven't had one since (at least) August 7. Is that correct? Thought this was supposed to come weekly?

**Comment by nelealbers** [comment]
@aversluis1 Maybe you have other uncompleted dialogs? Have you checked if that is the case by just typing 'verder'?

**Comment by aversluis1** [comment]
It doesn't appear like I have, so I am very curious about what is the problem

**Comment by aversluis1** [comment]
@emeijerperfectfit , can you share your experiences on this topic as well?

**Comment by emeijerperfectfit** [comment]
I did not get the weekly reflection since August 14th, but on that day I stopped the weekly reflection halfway.
Since then I've received only 1 reminder to continue, which was sent on the same day

**Comment by wbaccinelli** [comment]
Yes, this was error that has been fixed [here](https://github.com/PerfectFit-project/virtual-coach-rasa/pull/160), but it has been deployed just a couple of weeks ago. 
@aversluis1 @emeijerperfectfit could you try to run the weekly reflection by typing '/EXTERNAL_weekly_reflection'? Upon the completion you should be back in track

**Comment by emeijerperfectfit** [comment]
I now got the week 5 dialog, so that seems ok. Should the next one come in a week or quicker?

**Comment by wbaccinelli** [comment]
@emeijerperfectfit it will be scheduled on the day of the week that you set in the profile creation. But, since the duration of the intervention has been shortened, you will receive the closing dialog.

**Comment by aversluis1** [comment]
In the second testing I am doing (with the sensor), I got the message I am now in my third week (and I think this is correct).
